erigon-lib/p2p/sentry/protocol.go

```diff
@@ -0,0 +1,84 @@
+package sentry
+
+import (
+	"github.com/erigontech/erigon-lib/gointerfaces/sentryproto"
+)
+
+func MinProtocol(m sentryproto.MessageId) sentryproto.Protocol {
+	for p := sentryproto.Protocol_ETH65; p <= sentryproto.Protocol_ETH68; p++ {
+		if ids, ok := ProtoIds[p]; ok {
+			if _, ok := ids[m]; ok {
+				return p
+			}
+		}
+	}
+
+	return -1
+}
+
+func ProtocolVersion(p sentryproto.Protocol) uint {
+	return uint(p + 65)
+}
+
+var ProtoIds = map[sentryproto.Protocol]map[sentryproto.MessageId]struct{}{
```

Comment:
I don't think we should be duplicating all of these mappings - it is a sign that the sentry multiplexer should be part of the `sentry` package

Reply:
hmm actually those mappings are in `eth/protocol` package (I thought they are in the sentry one). but am curious could we reuse that existing `eth.FromProto` mapping

Reply:
and also move multiplexer in the p2p/sentry package

Reply:
discussed both offline:
- idea behind this is that Mark plans to move sentry to erigon-lib so putting multiplexer there as a stepping stone
- eth.FromProto uses its own invented values of the integers (65,66,67,...) instead of using the ones from the protobuf one (0,1,2,3,..)